homeassistant/components/tesla/switch.py

```diff
@@ -20,9 +20,34 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
             entities.append(RangeSwitch(device, coordinator))
         elif device.type == "sentry mode switch":
             entities.append(SentryModeSwitch(device, coordinator))
+        elif device.type == "heated steering switch":
+            entities.append(HeatedSteeringWheelSwitch(device, coordinator))
     async_add_entities(entities, True)
 
 
+class HeatedSteeringWheelSwitch(TeslaDevice, SwitchEntity):
+    """Representation of a Tesla Heated Steering Wheel switch."""
+
+    async def async_turn_on(self, **kwargs):
+        """Send the on command."""
+        _LOGGER.debug("Turn on Heating Steering Wheel: %s", self.name)
+        await self.tesla_device.set_steering_wheel_heat(True)
+        self.async_write_ha_state()
+
+    async def async_turn_off(self, **kwargs):
+        """Send the off command."""
+        _LOGGER.debug("Turn off Heating Steering Wheel: %s", self.name)
+        await self.tesla_device.set_steering_wheel_heat(False)
+        self.async_write_ha_state()
+
+    @property
+    def is_on(self):
+        """Get whether the switch is in on state."""
+        if self.tesla_device.get_steering_wheel_heat() is None:
+            return None
+        return self.tesla_device.get_steering_wheel_heat()
```

Comment:
```suggestion
        return self.tesla_device.get_steering_wheel_heat()
```

Reply:
I was was following the convention of the switch component here.

I agree that its not needed, but as I'm not the code owner, I didn't want to change their convention. It may have been for a reason I'm not aware of.

Reply:
`None` should only be checked if you are going to do something different (eg. convert to boolean, or format string).

I believe this should be cleaned up in the other sections (maybe in separate PR) but maybe wait for a second opinion...

Reply:
I can't remember why we had the check before. Feel free to remove it. It's probably vestigial now as the component has been around for a while now.

Reply:
I've accepted the suggestion, ran the code locally, and all works fine.

For history's sake, i have a Model 3 2021.